tests/requirements.in

```diff
@@ -1,7 +1,11 @@
+ansible<=2.6
```

Comment:
Why this bound?

Reply:
Because I use the ansible python API to parse inventory. But ansible APi is not considered as stable. That's the main reason I don't include metalk8s requirements. (but ok == 2.6 might be better)